rpc/Base.ts

```diff
@@ -102,32 +106,27 @@ export abstract class Client<
       method: methodName,
       params,
     };
+    const sendResult = await this.send(init);
+    if (sendResult instanceof Error) {
+      return sendResult;
+    }
     const isCorrespondingRes = IsCorrespondingRes(init);
     const ingressMessagePending = deferred<IngressMessage | InternalError>();
-    let stopListening: () => void = () => {};
     this.listen(
```

Comment:
There seem to be a microtask scheduling issue (that I haven't been able to identify).
The workaround is to schedule message/error handling as a task (with `setTimeout`).

I noticed that, without `setTimeout`, these 2 code snipped misses the first event for a subscription
```
const sub = await this.send(...)
this.listen((...) => {
  // do something with sub.result
})
```
```
const subPromise = this.send(...)
let sub
this.listen((...) => {
  // do something with sub.result
})
sub = await subPromise
```